research/picai/fl_nnunet/nnunet_client.py

```diff
@@ -521,3 +537,72 @@ def update_before_epoch(self, epoch: int) -> None:
     def get_client_specific_logs(self) -> Tuple[str, List[Tuple[LogLevel, str]]]:
         lr = self.optimizers["global"].param_groups[0]["lr"]
         return f" Current LR: {lr}", []
+
+    def get_properties(self, config: Config) -> Dict[str, Scalar]:
+        """
+        Return properties (sample counts and nnunet plans) of client.
+
+        If nnunet plans are not provided by the server, creates a new set of
```

Comment:
Great functionality!

Reply:
It's a little hacky right now due to mypy complaints. I'd like to get flwr to support something more official so we don't have to have this in our repo